Content.Server/GameObjects/Components/Destructible/Thresholds/Triggers/ITrigger.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Content.Server.GameObjects.EntitySystems;
+using Content.Shared.GameObjects.Components.Damage;
+using Robust.Shared.Interfaces.Serialization;
+
+namespace Content.Server.GameObjects.Components.Destructible.Thresholds.Triggers
+{
+    public interface ITrigger : IExposeData
```

Comment:
Same here, honestly. Would be best if it had a name that makes it clear this relates to destructible thresholds and stuff.

Reply:
Fixed in https://github.com/space-wizards/space-station-14/pull/3080/commits/c99be52304001825c5c0259f156f46fbca067c2d. See https://github.com/space-wizards/space-station-14/pull/3080#discussion_r570507140